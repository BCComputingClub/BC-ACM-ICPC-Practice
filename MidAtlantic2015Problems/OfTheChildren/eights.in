8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1021
2 4 500
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1021
2 4 550
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1000
0 3 1013
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1000
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1025
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1051
2 4 5000
2 5 997
4 6 991
5 7 983
6 7 977
-1 -1 -1
8
1000
1000
1000
1000
500
500
0 1 1009
0 3 1013
1 2 1019
3 4 1021
2 4 5000
2 5 997
4 6 991
5 7 950
6 7 978
-1 -1 -1
0

