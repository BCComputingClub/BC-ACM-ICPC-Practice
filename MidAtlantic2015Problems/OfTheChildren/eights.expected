1502
1028
1046
1499
1489
1508
1508
1475
